pkg/kuttlctl/cmd/test.go

```diff
@@ -151,6 +152,17 @@ For more detailed documentation, visit: https://kudo.dev/docs/testing`,
 			if len(options.TestDirs) == 0 {
 				return errors.New("no test directories provided, please provide either --config or test directories on the command line")
 			}
+			var APIServerArgs []string
+			var err error
+			if mockControllerFile != "" {
+				APIServerArgs, err = testutils.ReadMockControllerConfig(mockControllerFile)
```

Comment:
I'd like to see a log that we're using a user provided config here, and which one, it's easy to overlook things like that when configuring kuttl for tests.

Reply:
@ANeumann82 there is already an stdout output of the exact configuration used at the point of starting the control plane... is that sufficient or were you looking for more than that?

Reply:
I would like to see the actual file name of the config file that is used, but we don't have the `*testing.T` here which means the logging would be a bit weird. 

I'm ok with the way it is.